tests/unit/blueprints/lib-test.js

```diff
@@ -6,8 +6,8 @@ let setupTestHooks = blueprintHelpers.setupTestHooks;
 let emberNew = blueprintHelpers.emberNew;
 let emberGenerate = blueprintHelpers.emberGenerate;
 
-const expect = require('ember-cli-blueprint-test-helpers/chai').expect;
-const dir = require('chai-files').dir;
+const { expect } = require('chai');
```

Comment:
This PR streamlines all `chai` and `chai-files` imports.
Now we only import from `chai` and `chai-files` directly.